ml-agents/mlagents/trainers/learn.py

```diff
@@ -160,6 +160,10 @@ def parse_command_line(argv: Optional[List[str]] = None) -> CommandLineOptions:
         "--cpu", default=False, action="store_true", help="Run with CPU only"
     )
 
+    parser.add_argument(
+        "--version", action="version", version=UnityEnvironment.API_VERSION
```

Comment:
Should this be `--api-version`? To differentiate from the toolkit version @chriselion

Reply:
I think we should print all of them - see https://github.com/Unity-Technologies/ml-agents/pull/2945